src/Components/Search/Components/Location.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import { InputBox, Input } from './style';
+import LocationModal from './LocationModal';
+
+function Location({ modalState, departureState, arrivalState, switchSpot }) {
+  const [spots, setSpots] = useState(null);
+  const [isModalOn, openModal, closeModal] = modalState;
+  const [departure, setDeparture] = departureState;
+  const [arrival, setArrival] = arrivalState;
+
+  useEffect(() => {
+    fetch('/data/spots.json')
+      .then(spots => spots.json())
+      .then(spots => setSpots(spots));
+  }, []);
+
+  return (
+    <>
+      <InputBox size="40%" onClick={e => openModal(e)}>
```

Comment:
```suggestion
      <InputBox size="40%" onClick={openModal}>
```
- inline 함수로 써주지 않더라도 같은 동작을 할 것 같습니다.
- 동일하게 되어 있는 다른 함수들도 다시 한번 살펴주세요!

Reply:
네 확인했습니다 !